Cargo.toml

```diff
@@ -96,6 +97,10 @@ fuel-core-wasm-executor = { version = "0.41.7", path = "./crates/services/upgrad
 fuel-core-xtask = { version = "0.0.0", path = "./xtask" }
 fuel-gas-price-algorithm = { version = "0.41.7", path = "crates/fuel-gas-price-algorithm" }
 
+# Proof system
+fuel-core-global-merkle-root-storage = { version = "0.41.6", path = "./crates/proof_system/global_merkle_root/storage" }
```

Comment:
this might be out of date, latest is 0.41.7 in master

Reply:
Right, thanks for spotting!

Reply:
Done in https://github.com/FuelLabs/fuel-core/pull/2722/commits/f5cc2e2eca2ee26fb2e7efeb54e2b45abdae0098